OasisDEX/oasis-borrow
# Make Tabs Generic and match up to Design System

# [Make Tabs Generic and match up to Design System](https://app.shortcut.com/oazo-apps/story/5025/replace-tabswitcher-vaulttabswitcher-etc-components-with-design-system-ready-equivalents)
  
## Changes 👷‍♀️
- Remove VaultTabSwitcher (vault manage)
- Remove TabSwitcher (marketing pages)
- Update UnderlineTab
- Create GeneralManageTabs component
  
## How to test 🧪
- Run storybook and review stories visually
- Compare Tab switcher on homepage with staging
- Compare Tab switcher on asset page with staging
- Compare Tab switcher on vault suggestions on overview page with staging
- Compare Tab switcher on vault management view like Borrow position with staging
- Confirm you can switch Tabs via by clicking on buttons inside Banner components within sections of manage vault
